refresh-data.sh

```diff
@@ -8,6 +8,17 @@
 
 set -e
 
+if django-admin inspectdb wagtailcore_page | grep "db_table = 'wagtailcore_page'" && $REFRESH_DB == false;
+    then 
+        echo 'Skipping DB refresh as db already exists'
+        exit 1
+    elif $REFRESH_DB == true;
+        then
+            continue
+    else
+        continue
```

Comment:
Both macOS's `bash` and `bash` in the `python` container complain about this test the use of `continue`:

```
./refresh-data.sh: line 15: ==: command not found
./refresh-data.sh: line 19: continue: only meaningful in a `for', `while', or `until' loop
```

This also seems superfluous, as if the original test for the database falls through, the script will continue regardless.

```suggestion
```